src/layouts/Authenticated.vue

```diff
@@ -12,14 +12,12 @@
           :routes="routes"
           :logo-route="logoRoute"
           class="sidebar--grid"
-          :key="currentUser && currentUser.permissions"
         />
         <Slide width="150" v-else>
           <NavMenu
             :routes="routes"
             :logo-route="logoRoute"
             class="flex flex-col text-sm"
-            :key="currentUser && currentUser.permissions"
```

Comment:
@raymondxd -- I think these key's were purposely being used in order to forcefully refresh the component... @tabiodun is this correct?

Reply:
Oops, sorry if that was added by intention. I removed these keys because key should be one of number or string. 
In the developer console, it was complaining that wrong type of key was used for them.
@tabiodun @BradenM So, if these keys are added by intention, I am good with adding them back but want to discuss the value to use for `:key` instead of using the user object directly.

Reply:
Sure, I'll let @tabiodun handle this one since he wrote it :)

Reply:
Yeah they are for permission refreshes basically

Reply:
@tabiodun thanks for your confirmation.
If so, do we need to use user object for `key` there? What happens if we use `user.id` for `key`?

Reply:
The issues it that navbar changes based on the user permissions, not the user id. So what we really want is the key to be `user.premissions` but if the user isn't available, it will error

Reply:
@tabiodun I will revert it back as it was but plz keep in mind that it will cause errors on developer console as `user.permissions` is an object.
I think it will be better for you to think of changing `user.permissions` into a string to use for `:key`.

Reply:
We can do `JSON.stringify(user && user.premissions)` which will be a string maybe

Reply:
Great. let's do it then

Reply:
@tabiodun Now it's updated as we discussed. Plz check.